test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -10986,6 +10989,33 @@ def forward(self, x):
         self.run_test(Module(False), x, rtol=1e-3, atol=1e-6)
         self.run_test(Module(True), x, rtol=1e-3, atol=1e-6)
 
+    @skipIfUnsupportedMinOpsetVersion(16)
+    def test_grid_sample(self):
+        n, c, h_in, w_in, h_out, w_out = 1, 1, 3, 2, 2, 4
+
+        class Module(torch.nn.Module):
+
+            def __init__(self, mode: str, padding_mode: str, align_corners: bool) -> None:
+                super().__init__()
+                self.mode, self.padding_mode, self.align_corners = mode, padding_mode, align_corners
+
+            def forward(self, input, grid):
+                return torch.nn.functional.grid_sample(input, grid, self.mode, self.padding_mode, self.align_corners)
+
+        for mode, padding_mode, align_corners in itertools.product(
+            ("bilinear", "nearest", "bicubic"),  # mode
+            ("zeros", "border", "reflection"),  # padding_mode
+            (True, False),  # align_corners
+        ):
+            # note (mkozuki): Skip the combinations that fail locally.
```

Comment:
I would make it more explicit that test cases are skipped. Possible to use some kind of skipping decorators?